src/RoadRegistry.BackOffice.ExtractHost/ZipArchiveWriters/ContourQueryExtensions.cs

```diff
@@ -0,0 +1,142 @@
+namespace RoadRegistry.BackOffice.ExtractHost.ZipArchiveWriters
+{
+    using System.Data;
+    using System.Data.SqlTypes;
+    using System.Linq;
+    using Editor.Schema.GradeSeparatedJunctions;
+    using Editor.Schema.RoadNodes;
+    using Editor.Schema.RoadSegments;
+    using Microsoft.Data.SqlClient;
+    using Microsoft.EntityFrameworkCore;
+    using NetTopologySuite.Geometries;
+    using NetTopologySuite.IO;
+
+    public static class ContourQueryExtensions
+    {
+        private static SqlParameter ToSqlParameter(this MultiPolygon contour)
```

Comment:
TIL: if you use `FromSqlRaw()` that nice `NetTopologySuite` integration you have going just became your problem to implement.